### DISCUSSION TITLE
Vote to retire @andymoe

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#103] [Content Team]
@paketo-buildpacks/content-maintainers - I'd like to propose to move @andymoe to an emeritus maintainer position per [his recent message](https://paketobuildpacks.slack.com/archives/C0229DVMFM5/p1642687487008200) about reduced time to contribute. In an effort to recognize him, he will receive an emeritus status recognition on https://github.com/paketo-buildpacks/community. He will then also be removed from his current position as an active maintainer to the Content team.

Can I get a 👍 / 👎 vote?

**Comment by sophiewigmore** [comment]
bump @fg-j @TisVictress

**Comment by andymoe** [comment]
😭
